english/mushroom/eat/eat_mushroom.md

```diff
@@ -1,4 +1,6 @@
-The mushroom makes great soup. You brought them back.
+The mushroom looks so great and they may make great soup. You brought them back.
+
 
 Do you:
-[Light a fire?] (../../light-fire/fire.md)
\ No newline at end of file
+[Light a fire?] (../../light-fire/fire.md)
+[eat them as raw?](../../super-man/save-the-word.md)
```

Comment:
There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.